internal/core/context.go

```diff
@@ -89,6 +87,13 @@ func ExtractBinaryName(ctx context.Context) string {
 	return extractMeta(ctx).BinaryName
 }
 
+func ExtractProfileName(ctx context.Context) string {
+	if extractMeta(ctx).ProfileFlag != "" {
+		return extractMeta(ctx).ProfileFlag
+	}
+	return ExtractEnv(ctx, "SCW_PROFILE")
```

Comment:
Should we use a constant available in the SDK since https://github.com/scaleway/scaleway-sdk-go/pull/400 ?